### ISSUE TITLE
breaks on java 1.4.2

### ISSUE THREAD

**Issue by danieldkim** [jezhumble/javasysmon#1]
looks like the code is using StringBuilder, which was introduced in 1.5:

```
$  /opt/WebSphere5/AppServer/java/bin/java SysMonTest
Exception in thread "main" java.lang.NoClassDefFoundError: java/lang/StringBuilder
        at com.jezhumble.javasysmon.FileUtils.slurpFromInputStream(FileUtils.java:69)
        at com.jezhumble.javasysmon.FileUtils.slurp(FileUtils.java:40)
        at com.jezhumble.javasysmon.FileUtils.runRegexOnFile(FileUtils.java:91)
        at com.jezhumble.javasysmon.LinuxMonitor.currentPid(LinuxMonitor.java:109)
        at com.jezhumble.javasysmon.JavaSysMon.currentPid(JavaSysMon.java:166)
        at SysMonTest.main(SysMonTest.java:7)
```

**Comment by jezhumble** [comment]
Thanks. I'll take a look at fixing this soon.

**Comment by jezhumble** [comment]
Fixed in 0.3.3
